app/models/spree/user_decorator.rb

```diff
@@ -18,10 +18,17 @@
   accepts_nested_attributes_for :ship_address
 
   attr_accessible :enterprise_ids, :enterprise_roles_attributes, :enterprise_limit, :bill_address_attributes, :ship_address_attributes
-  after_create :send_signup_confirmation
 
   validate :limit_owned_enterprises
 
+  # We use the same options as Spree and add :confirmable
+  devise :database_authenticatable, :token_authenticatable, :registerable, :recoverable,
+    :rememberable, :trackable, :validatable, :confirmable, :encryptable,
+    :reconfirmable => true, confirmation_keys: [ :id, :email ], :encryptor => 'authlogic_sha512'
```

Comment:
Do we need to use the `:trackable` flag? That's for logging IP addresses and number of sign-ins right?